CMakeLists.txt

```diff
@@ -7,8 +7,6 @@ include(GNUInstallDirs)
 
 # find dependencies
 set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake ${CMAKE_MODULE_PATH})
-find_package(MPI REQUIRED)
```

Comment:
Was this causing errors? I guess we explicitly use Kokkos and MPI in this code so is it best practice to include them separately, even though we would also get them from Cabana?

Reply:
No errors, just trying to simplify and match what I have in CabanaMD. I believe this was @junghans' suggestion

Reply:
If you are using MPI directly keep it as Cabana could be build without MPI or check Cabana was build with MPI.

Reply:
I updated to check Cabana has both MPI and Cajita. I prefer this to force versions of MPI to match in Cabana and ExaMPM, but if you disagree we can close